src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -3443,7 +3443,24 @@ private void GetContainerPropertiesInternal()
                     //
                     if (RuntimeId == Guid.Empty)
                     {
-                        ContainerObRoot = (string)computeSystemPropertiesType.GetProperty("ObRoot").GetValue(computeSystemPropertiesHandle);
+                        var obRootfieldInfo = computeSystemPropertiesType.GetField("ObRoot");
```

Comment:
Please add similar comment about Hyper-V team changing property to field, to make it clear to others maintaining this code.